scio-core/src/main/java/com/spotify/scio/transforms/JavaAsyncDoFn.java

```diff
@@ -18,18 +18,18 @@
 package com.spotify.scio.transforms;
 
 import com.google.common.collect.Iterables;
-import org.apache.beam.sdk.transforms.DoFn;
-
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 import java.util.function.Function;
+import org.apache.beam.sdk.transforms.DoFn;
```

Comment:
There's no actual import change. IntelliJ auto formatting?

Reply:
yeah it was